### PR TITLE
docs: updated readme file

### PR DESCRIPTION
**Pull request by yevhen-burkovskyi**

### CODE REVIEW

**Review by DraugS — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,40 @@
-# Template project
-
-> Template for [Axone](https://axone.xyz) opensource projects.
+# Hub backend
 
 [![lint](https://img.shields.io/github/actions/workflow/status/axone-protocol/template-oss/lint.yml?branch=main&label=lint&style=for-the-badge&logo=github)](https://github.com/axone-protocol/template-oss/actions/workflows/lint.yml)
```
Comment (ccamel) [inline README.md:3]: Please, update the link to point to this repository.

Comment (ccamel) [inline README.md:3]: By the way, you could take the opportunity of this PR to add the status badge for the build.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+- [npm](https://www.npmjs.com/) (comes with Node.js) or [Yarn](https://yarnpkg.com/) (~1.22.19)
+
+## Installation
+
+1. **Clone the repository:**
+   ```bash
+   git clone https://github.com/axone-protocol/hub-backend.git
+2. **Install dependencies:**
+    ```bash
+    npm install
```
Comment (ccamel) [inline README.md:23]: I woudn't encourage mixing the use of `npm` and `yarn` as they use different locking file formats to manage dependencies. So I suggest only mentioning `yarn` as the only package manager.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,40 @@
...
+    # or
+    yarn install
+3. **Add docker containers**
```
Comment (ccamel) [inline README.md:26]:
```suggestion
3. **Run docker containers**
```

And explain the intention (and necessity) of doing so.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,40 @@
...
+    ```bash
+    docker run -d --name your_db_container_name -p 5432:5432 -e POSTGRES_PASSWORD=your_password POSTGRES_DB=your_db_name timescale/timescaledb-ha:pg16
+    docker run --name your_redis_container_name -p 6379:6379 -d redis
+4. **Configure env**
+    After creation .env.local change env variable to what you settup in docker
```
Comment (ccamel) [inline README.md:31]: Please correct the wording and typos.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,40 @@
...
+    ```bash
+    cp .env.example .env.local
+5. **Starting server**
+    ```bash
+    yarn dev for dev mode
+    yarn start for live mode
```
Comment (ccamel) [inline README.md:37]:
```suggestion
    yarn dev # for dev mode
    yarn start # for live mode
```

**Review by ccamel — COMMENTED** [review]
A few comments which I would ask you to take into consideration.

**Comment by ccamel** [comment]
@yevhen-burkovskyi, please don't merge PRs that don't pass the CI. The linter has [reported errors](https://github.com/axone-protocol/hub-backend/actions/runs/9716215563/job/26819341691?pr=51) that need to be dealt with. Thanks.

**Pull request merged**